static/dbstructure.config.php

```diff
@@ -289,7 +289,8 @@
 		"fields" => [
 			"id" => ["type" => "int unsigned", "not null" => "1", "extra" => "auto_increment", "primary" => "1", "comment" => ""],
 			"name" => ["type" => "varchar(96)", "not null" => "1", "default" => "", "comment" => ""],
-			"url" => ["type" => "varbinary(255)", "not null" => "1", "default" => "", "comment" => ""]
+			"url" => ["type" => "varbinary(255)", "not null" => "1", "default" => "", "comment" => ""],
+			"type" => ["type" => "tinyint unsigned", "comment" => "Type of the tag (Unknown, Collection or Account)"],
```

Comment:
It seems to me there are 5 tag types: Account, Collection, General, Follower and Unknown from the Tag constants created. Shouldn't this comment reflecting this?